### ISSUE TITLE
Adding health_check to a load balancer service throws `name 'LoadBalancerServiceHealthCheck' is not defined`

### ISSUE THREAD

**Issue by te-online** [ansible-collections/hetzner.hcloud#68]
##### SUMMARY

I have this task to add a service to an existing Load Balancer:

```yaml
- name: Add Load Balancer service
  hetzner.hcloud.hcloud_load_balancer_service:
    load_balancer: "lb.{{ domain_name }}"
    listen_port: 443
    destination_port: 8080
    protocol: https
    http:
      certificates: ["{{ cert_name }}"]
      redirect_http: yes
    health_check:
      http:
        path: "/health"
        status_codes: ["2??", "3??"]
      protocol: http
      port: 8080
      interval: 15
      timeout: 10
      retries: 3
    state: present
```

This works fine if I remove the `health_check` node. I tried several different configurations for the health check, but the error I'm getting is always the same: 

```
NameError: name 'LoadBalancerServiceHealthCheck' is not defined
```

Now, if I check the [corresponding file](https://github.com/ansible-collections/hetzner.hcloud/blob/master/plugins/modules/hcloud_load_balancer_service.py#L417), it looks like it is indeed defined. Maybe it is some sort of import of definition error?

I'm not very good at reading python code, but I can't seem to spot any error and I've double checked my task configuration in accordance with the docs as well.

That's why I unfortunately can't provide any initial ideas of why and where this fails.

**Full error output**
<details>

```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: NameError: name 'LoadBalancerServiceHealthCheck' is not defined
fatal: [localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/root/.ansible/tmp/ansible-tmp-1616688997.9039645-115-258734497133304/AnsiballZ_hcloud_load_balancer_service.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/root/.ansible/tmp/ansible-tmp-1616688997.9039645-115-258734497133304/AnsiballZ_hcloud_load_balancer_service.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/root/.ansible/tmp/ansible-tmp-1616688997.9039645-115-258734497133304/AnsiballZ_hcloud_load_balancer_service.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.hetzner.hcloud.plugins.modules.hcloud_load_balancer_service', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/local/lib/python3.8/runpy.py\", line 207, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/local/lib/python3.8/runpy.py\", line 97, in _run_module_code\n    _run_code(code, mod_globals, init_globals,\n  File \"/usr/local/lib/python3.8/runpy.py\", line 87, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload_33fmge0u/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload.zip/ansible_collections/hetzner/hcloud/plugins/modules/hcloud_load_balancer_service.py\", line 616, in <module>\n  File \"/tmp/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload_33fmge0u/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload.zip/ansible_collections/hetzner/hcloud/plugins/modules/hcloud_load_balancer_service.py\", line 610, in main\n  File \"/tmp/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload_33fmge0u/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload.zip/ansible_collections/hetzner/hcloud/plugins/modules/hcloud_load_balancer_service.py\", line 493, in present_load_balancer_service\n  File \"/tmp/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload_33fmge0u/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload.zip/ansible_collections/hetzner/hcloud/plugins/modules/hcloud_load_balancer_service.py\", line 373, in _create_load_balancer_service\n  File \"/tmp/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload_33fmge0u/ansible_hetzner.hcloud.hcloud_load_balancer_service_payload.zip/ansible_collections/hetzner/hcloud/plugins/modules/hcloud_load_balancer_service.py\", line 417, in __get_service_health_checks\nNameError: name 'LoadBalancerServiceHealthCheck' is not defined\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}
```

</details>

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

hetzner.hcloud.hcloud_load_balancer_service

##### ANSIBLE VERSION

```paste below
ansible 2.9.10
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.8/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.3 (default, Jun  9 2020, 17:49:41) [GCC 8.3.0]
```

##### CONFIGURATION

```paste below
empty
```

##### OS / ENVIRONMENT

Docker image `python:3.8-slim`

##### STEPS TO REPRODUCE

Use the task from summary section to reproduce the issue.

##### EXPECTED RESULTS

Load Balancer receives service with health check configured or existing service is updated with health check config.

##### ACTUAL RESULTS

See above error description

**Comment by ieugen** [comment]
I can confirm this behavior.
I followed instructions from https://docs.ansible.com/ansible/latest/collections/hetzner/hcloud/hcloud_load_balancer_service_module.html#parameter-health_check .

Same error.

**Comment by LKaemmerling** [comment]
Hey @te-online & @ieugen,

thank you for the report. I fixed it with e2dfd88 and I will release a new version within the next couple of minutes.

**Comment by LKaemmerling** [comment]
I just released v1.4.0. It will be available in the next couple minutes.
